app/assets/javascripts/controllers/live_metrics/util/metrics-parse-url-factroy.js

```diff
@@ -4,10 +4,12 @@ angular.module('miq.util').factory('metricsParseUrlFactory', function() {
     var pathname = $window.location.pathname.replace(/\/$/, '');
     dash.providerId = '/' + (/^\/[^\/]+\/([r\d]+)$/.exec(pathname)[1]);
 
-    // TODO: get this values from GET/POST values ?
+    // parse window's url
+    dash.params = (new URL($window.location)).searchParams;
```

Comment:
You'll have to put the custom parser back :(

`URL` is not supported in edge, nor IE11.

And yes, I'm fine with the custom parser as it was done before, we'll need to fix it but given such code already exists in various controllers, this will have to be a larger refactoring effort, so .. one more won't hurt too much :).